axon/src/message.rs

```diff
@@ -229,48 +193,3 @@ pub struct MsgTimeoutPacket {
     pub next_sequence_recv: U256,
     pub proofs: Proofs,
 }
-
-#[cfg(test)]
-mod tests {
-    use crate::message::Envelope;
-
-    use super::MsgType;
-    use super::Vec;
-
-    #[test]
-    fn msg_type_encode_decode() {
-        let mut types = Vec::new();
-        types.push(MsgType::MsgClientCreate);
-        types.push(MsgType::MsgClientUpdate);
-        types.push(MsgType::MsgClientMisbehaviour);
-
-        types.push(MsgType::MsgConnectionOpenInit);
-        types.push(MsgType::MsgConnectionOpenTry);
-        types.push(MsgType::MsgConnectionOpenAck);
-        types.push(MsgType::MsgConnectionOpenConfirm);
-
-        types.push(MsgType::MsgChannelOpenInit);
-        types.push(MsgType::MsgChannelOpenTry);
-        types.push(MsgType::MsgChannelOpenAck);
-        types.push(MsgType::MsgChannelOpenConfirm);
-        types.push(MsgType::MsgChannelCloseInit);
-        types.push(MsgType::MsgChannelCloseConfirm);
-
-        types.push(MsgType::MsgSendPacket);
-        types.push(MsgType::MsgRecvPacket);
-        types.push(MsgType::MsgWriteAckPacket);
-        types.push(MsgType::MsgAckPacket);
-        types.push(MsgType::MsgTimeoutPacket);
-
-        for i in 1..types.len() {
```

Comment:
Removed these tests because they have off-by-one errors and are no longer necessary.